SharperUniverse/Core/Builder/GameBuilder.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System.Reflection;
-using Serilog;
```

Comment:
Uhhh why was this removed?

Reply:
I have no clue. Perhaps rider auto removed it. Is anything from that namespace used?

Reply:
Yes - at least it should be. Its our logging tool.